scanner/src/funTest/kotlin/HttpStorageTest.kt

```diff
@@ -52,7 +52,7 @@ import java.time.Instant
 
 import kotlin.random.Random
 
-class HttpCacheTest : StringSpec() {
+class HttpStorageTest : StringSpec() {
```

Comment:
Is this actually intended? asking because its not closely related to the ScanResultsCache

Reply:
Still, this test actually "mocks" an ArtifactoryStorage, so I believe renaming it is consequent.